libs/ledgerjs/packages/hw-app-btc/src/BtcNew.ts

```diff
@@ -297,6 +297,38 @@ export default class BtcNew {
     return serializedTx.toString("hex");
   }
 
+  /**
+   * Signs an arbitrary hex-formatted message with the private key at
+   * the provided derivation path according to the Bitcoin Signature format
+   * and returns v, r, s.
+   */
+  async signMessage({
```

Comment:
Could you please consider covering this function with unit tests?
For simplicity you can create a new, dedicated test file.

Reply:
This calls signMessage from `appClient`, so not sure there's an easy way of unit-testing this without setting up speculos (e.g. see the [tests in the js client of the bitcoin app](https://github.com/LedgerHQ/app-bitcoin-new/blob/develop/bitcoin_client_js/src/__tests__/appClient.test.ts)).

Reply:
Good point, nevermind.